src/main/java/com/hubspot/jinjava/lib/filter/RandomFilter.java

```diff
@@ -75,20 +74,20 @@ public Object filter(Object object, JinjavaInterpreter interpreter, String... ar
       if (size == 0) {
         return null;
       }
-      int index = ThreadLocalRandom.current().nextInt(size);
+      int index = interpreter.getRandom().nextInt(size);
       while (index-- > 0) {
         it.next();
       }
       return it.next();
     }
     // number
     if (object instanceof Number) {
-      return ThreadLocalRandom.current().nextLong(((Number) object).longValue());
+      return interpreter.getRandom().nextInt(((Number) object).intValue());
```

Comment:
There's no `nextLong()` that takes a bound in `java.util.Random`. I figured random ints are probably big enough.